pylint/pyreverse/main.py

```diff
@@ -77,6 +79,7 @@
         dict(
             short="A",
             default=None,
```

Comment:
If this a boolean if could be 
```suggestion
            default=False,
```
, right ?

Reply:
See previous discussion. We actually use the `None` value here to see if the option was explicitly set to `True` or `False`. If it was not set explicitly we determine the value based on another option's value.
It's a bit convoluted, but changing it would be a breaking change as default behaviour would change.